### DISCUSSION TITLE
STM32U0 Support

### DISCUSSION THREAD

**Discussion by FranzForstmayr** [modm-io/modm#1199] [Ideas]
Hi,

is there a plan to support the stm32u0 devices, especially that one found on the [STM32U083C-DK](https://www.st.com/en/evaluation-tools/stm32u083c-dk.html#overview), a stm32u083mct6.
If there's anything I can do to help for this issue let me know.

Thanks and BR
Franz

**Comment by salkinium** [comment]
Yes, any new STM32 famliy is welcome and probably not too difficult to add considering the significant overlap of existing peripheral drivers.

It's probably best to check the STM32C0 support PR for what needs to be done: https://github.com/modm-io/modm/pull/1188 though I suspect that the U0 is more similar to the L0.

In this case, you first need to enable STM32U0 support in modm-devices: https://github.com/modm-io/modm-devices/
It might be fairly simple though: https://github.com/modm-io/modm-devices/pull/82
